lib/library.go

```diff
@@ -349,6 +350,22 @@ func SendTransaction(txArgsJSON, password *C.char) *C.char {
 	return C.CString(prepareJSONResponseWithCode(hash.String(), err, code))
 }
 
+// SignTypedData unmarshall data into TypedData, validate it and signs with selected account,
+// if password matches selected account.
+//export SignTypedData
+func SignTypedData(data, password *C.char) *C.char {
```

Comment:
Shouldn't it be exposed through JSON-RPC somehow? The EIP spec mentions `eth_signTypedData`. Should it be public?

Reply:
i dont think so. afair we handle signing differently. and there must be a part in status-react that will provide api for dapps. but this is pretty much the same as signing of transactions or unstructured data.

Reply:
Ok, that's fine as long as everyone is aware how it will be integrated.